gridt/src/app/core/settings.service.ts

```diff
@@ -87,120 +94,35 @@ export class SettingsService {
    */
   private storeLocalSettings(settings: Settings): void {
     console.log(`Storing ${JSON.stringify(settings)} into the local storage.`);
-    this.secStore.set$("settings", settings);
-  }
-
-  /**
-   * Function to process server response
-   */
-  public getSettingsFromServer(): void {
-    console.log("Populating secure storage.");
-    forkJoin({
-      server: this.getServerIdentity$
-    }).pipe(
-      tap( console.log ),
-      map( (settings) => {
-        console.log(`received server settings ${JSON.stringify(settings.server)}`);
-        return {...settings.server}
-        // Server settings have priority over local settings
-      })
-    ).subscribe( (set) => this.secStore.set$("settings", {set}).subscribe() );
+    this.secStore.set$("settings", settings).subscribe();
   }
 
   /**
    * Update the _user_settings$ by combining Local and Server responses.
+   *
+   * While a forkJoin may be usefull, it doesn't allow for simple error
+   * catching, hence the two seperate subscriptions for local and server.

```

Comment:
I am not yet entirely convinced, why could you not catch the errors before they enter the forkJoin?

Reply:
You have done so successfully for the secureStorage which seems to work quite well.